glean_parser/go_server.py

```diff
@@ -32,7 +35,15 @@
 
 # Adding a metric here will require updating the `generate_metric_type` function
 # and require adjustments to `metrics` variables the the template.
-SUPPORTED_METRIC_TYPES = ["string", "quantity", "event", "datetime"]
+SUPPORTED_METRIC_TYPES = ["string", "quantity", "event", "datetime", "boolean"]
+
+
+def generate_ping_type_name(ping_name: str) -> str:
+    return f"Ping{util.Camelize(ping_name)}"
```

Comment:
I did consider that, but since the "Event" in generate_event_type_name was a prefix I followed that. Agree that it would read better as a suffix so let me try that out for all three of these.